Controller/CRUDController.php

```diff
@@ -1011,8 +1011,8 @@ protected function configure()
 
         $rootAdmin = $this->admin;
 
-        if ($this->admin->isChild()) {
-            $this->admin->setCurrentChild(true);
+        while ($rootAdmin->isChild()) {
```

Comment:
I tried to see what would happen if two admins are both child of each other, but I failed. The container simply runs out of memory at its initialization (w/ SF 3.3.4).